package.json

```diff
@@ -27,39 +27,40 @@
     "lwm2mAgent": "bin/lwm2mAgent.js"
   },
   "engines": {
-    "node": ">=0.10.8"
+    "node": ">=4.8.4"
```

Comment:
This should be annotated in CHANGES_NEXT_RELEASE, for instance:

> FEATURE update node version to 4.8.4

Reply:
Fixed in 34627c7e1c30ce1820fdded1534d77af8bb6f2c5